examples/expression-oriented-style/expression_oriented_style.md

```diff
@@ -0,0 +1,7 @@
+Ballerina supports statements for familiarity but also tries to enable an expression-oriented style to be more intuitive.
+
+Here are few things that you can do using expressions in Ballerina.
```

Comment:
The way we've structured it now, the description needs to make sense on its own. Since this kinda incomplete we need to have examples here.